dandiapi/api/services/metadata/__init__.py

```diff
@@ -90,7 +92,9 @@ def version_aggregate_assets_summary(version: Version) -> None:
 
     assets_summary = aggregate_assets_summary(
         asset.full_metadata
-        for asset in version.assets.filter(status=Asset.Status.VALID)
+        for asset in version.assets.filter(
+            Q(status=Asset.Status.VALID) | Q(zarr__status=ZarrArchiveStatus.UPLOADED)
```

Comment:
What was the problem that led to this particular change? We need @jjnesbitt to confirm, but I don't think we should be counting Zarr archives in the `UPLOADED` state (those have not yet been "finalized" in the terminology of the Zarr API; once they are, I believe they move into the `COMPLETE` state).

Reply:
Good catch -- updated to `COMPLETE` here and in the unit test.

The expansion of the `filter` query here is to match similar logic in the `validate_pending_asset_metadata` Celery task -- [specific code reference here](https://github.com/dandi/dandi-archive/blob/0d4f6038b1a208617716eaf68859ea6b157a4eb3/dandiapi/api/tasks/scheduled.py#L72-L74)